nixnet_examples/can_frame_queued_io.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pprint
+import six
+import time
+
+from nixnet import constants
+from nixnet import nx
+from nixnet import types
+
+
+pp = pprint.PrettyPrinter(indent=4)
+
+
+def main():
+    database_name = 'NIXNET_example'
```

Comment:
Yes, this requires the NIXNET_example database to be uninstalled on the system.
I believe it gets installed by default with driver - @XNET-PSE Jeff, can you confirm?